app/scripts/modules/core/src/help/help.contents.ts

```diff
@@ -86,7 +86,10 @@ module(HELP_CONTENTS, [])
         <p>If you want to start from scratch, select "None".</p>
         <p>You can always edit the cluster configuration after you've created it.</p>`,
     'pipeline.config.expectedArtifact.matchArtifact': `
-        <p>This specifies which fields in your incoming artifact to match against. Every field that you supply will be used to match against all incoming artifacts. If all fields specified match, the incoming artifact is bound to your pipeline context.</p>
+        <p>
+          This specifies which fields in your incoming artifact to match against. Artifact fields are populated by the jinja transform defined as part of your subscription's configuration.
```

Comment:
It's possible the artifacts were present before the payload was transformed via jinja, or that no transformation was applied.

Reply:
Ah ok, I've added some conditional phrasing around that.